plugins/dialog/plugin.js

```diff
@@ -3034,16 +3034,33 @@ CKEDITOR.DIALOG_STATE_BUSY = 2;
 		CKEDITOR.VALIDATE_OR = 1;
 		CKEDITOR.VALIDATE_AND = 2;
 
+		/**
+		 * The namespace with dialog helper validation functions.
+		 *
+		 * @class
+		 * @singleton
+		 */
 		CKEDITOR.dialog.validate = {
+			/**
+			 * Performs validation functions composition.
+			 *
+			 * ```
+			 * CKEDITOR.dialog.validation.functions(
+			 * 	CKEDITOR.dialog.validation.notEmpty( 'Value is required.' ),
```

Comment:
Incorrect doc block type (we use indentation-based ones) and code indentation (should be indented using <kbd>Tab</kbd>).